devmand/gateway/test_cfg/localhost.json

```diff
@@ -0,0 +1,14 @@
+{
+  "openconfig-interfaces:interfaces": {
+    "interface": [
+      {
+        "name": "0/11",
+        "config": {
+          "type": "iana-if-type:ethernetCsmacd",
+          "name": "0/11",
+          "enabled": true
+        }
+      }
+    ]
+  }
+}
```

Comment:
nitpick but can we have `\n` at end of each file? In CLion prefs search for `Ensure line feed at file end`

Reply:
yes I can add it.

As for \n I am not sure, I took the delimiter from Java. I’ll look at it on thursday, maybe we can add a flag to distinguish between KA enter and multi command delimiter